geode-core/src/main/java/org/apache/geode/distributed/internal/membership/gms/membership/GMSJoinLeave.java

```diff
@@ -729,7 +729,25 @@ private void recordViewRequest(DistributionMessage request) {
       logger.warn("unable to record a membership view request due to this exception", t);
       throw t;
     }
+  }
+
+  private void sendDHKeys() {
+    if (isCoordinator
+        && !services.getConfig().getDistributionConfig().getSecurityUDPDHAlgo().isEmpty()) {
```

Comment:
What does the condition `!services.getConfig().getDistributionConfig().getSecurityUDPDHAlgo().isEmpty()` indicate?